migration/migrate.go

```diff
@@ -0,0 +1,47 @@
+package migration
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+	"os"
+)
+
+// Execute migrations.
+// Use "migrate-up" argument after run of program. It will create new table
+// Or use "migrate-down" - that will remove table
+// Example: go run main.go migrate-up
+func Migrate(db *sql.DB) error {
+	for _, arg := range os.Args {
+		if arg == "migrate-up" {
+			log.Println("Migration starts up")
```

Comment:
Shall we use our new logger here as well?

Reply:
Agreed